### PR TITLE
Small fixes on infopage design

### PR DESCRIPTION
**Pull request by norbye**
# Description

Key points:
* Prevent overflow
* Fill available space
* Add more responsive breakpoints.
* Prevent padding on top caused by a not specific enough selector

# Result

It requires a loot of pictures to showcase every edge case of this change, so to check every screen size you'd rather have to pull this and test it locally.

Before: unnecessary padding on top + overflow on the right side
image

After
image

Before: doesn't fill the entire page
image

After: equal paddings on the right and left side
image

Before: the breakpoints are either desktop or mobile, where mobile is a single row even if its pretty wide
image

After: three breakpoints, with 3, 2 and 1 columns depending on the screen size
image

# Testing

- [x] I have thoroughly tested my changes.

Please describe what and how the changes have been tested, and provide instructions to reproduce if necessary.

### CODE REVIEW

File: app/routes/pages/components/subcomponents/TextWithTitle/index.tsx [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
       <p className={styles.text}>{text}</p>
     </div>
   );
 };
 
-const TextWithTitle = ({ title, text, extraStyle }: Props) => {
+const TextWithTitle = ({ title, text, extraStyle, extraClassName }: Props) => {
   return (
-    <div className={styles.container} style={extraStyle}>
+    <div className={`${styles.container} ${extraClassName}`} style={extraStyle}>
```
Comment (LudvigHz) [inline app/routes/pages/components/subcomponents/TextWithTitle/index.tsx:28]: `cx`?

**Review by LudvigHz — APPROVED** [review]
Awesome! Much needed👌👌

**Review by ivarnakken — APPROVED** [review]
Yay💯💯

File: app/routes/pages/components/subcomponents/TextWithTitle/index.tsx (hunk 1 above)
Comment (norbye) [reply app/routes/pages/components/subcomponents/TextWithTitle/index.tsx]: Ah yes, was testing moving back and forth to see if the css selectors specificity would change (it didn't:/) - moved back to cx now
